app/api/bookyt/api/bookings.rb

```diff
@@ -0,0 +1,31 @@
+module Bookyt
+  class API
+    class Bookings < Grape::API
+      resource :bookings do
+        desc 'Create a new booking'
+        params do
+          requires :title, type: String, desc: 'Title of the booking'
+          requires :amount, type: Float, desc: 'Amount to be booked'
+          requires :value_date, type: Date, desc: 'Date of the booking'
+          requires :credit_account_tag, type: String, desc: 'Tag of the credit account'
+          requires :debit_account_tag, type: String, desc: 'Tag of the debit account'
+          optional :comments, type: String, desc: 'Additional comments'
+        end
+        post do
+          credit_account = Account.tagged_with(params[:credit_account_tag]).first
+          debit_account = Account.tagged_with(params[:debit_account_tag]).first
+          attributes = {
+            title: params[:title],
+            amount: params[:amount],
+            value_date: params[:value_date],
+            credit_account: credit_account,
+            debit_account: debit_account,
+            comments: params[:comments],
+          }
+          booking = Booking.create!(attributes)
```

Comment:
When we get a validation error (like no Account has been found by tag), we get a Rails validation error like the following:

```
{"status":422,"message":"Soll muss ausgef\u00fcllt werden und Haben muss ausgef\u00fcllt werden","errors":{"debit_account":["muss ausgef\u00fcllt werden"],"credit_account":["muss ausgef\u00fcllt werden"]}}
```

This is a bit inconsistent as we now have two layers of validation messages: Active Model and Drape. Would be great if we could only return Drape style validations.

It also shows an issue with translated error messages. We might need to set the locale to `en` for API requests.